doc/setup.md

```diff
@@ -16,21 +17,50 @@ When installing SkyPortal on Debian-based systems, 2 additional packages are req
 - libcurl4-gnutls-dev
 - libgnutls28-dev
 
-## Source download, Python environment
+## Package Managers
```

Comment:
so uv and bun are package managers but also virtualenv & javascript runtimes respectively. Maybe you can have one section for each as they are not just package managers. One section for each is also a nice way to explain how we get the python part to work, and the js part. Might want to have some little text there saying that the python part is for the backend, and js for the frontend. Could also be still one section but with 2 sub sections?

Reply:
Ok! I like the idea of one section, perhaps titled `Development Setup`, and then sub sections for BE/FE.